tests/tests_integration/test_api/test_iam.py

```diff
@@ -26,18 +27,22 @@ def service_account_id(cognite_client):
 
 
 class TestAPIKeysAPI:
+    @pytest.mark.skip("most API keys methods failing, probably due to deprecation")
```

Comment:
@erlendvollset has the sdk test tenant been moved to oidc?

Reply:
yup.